process/block/interface.go

```diff
@@ -45,16 +45,6 @@ type crossNotarizer interface {
 	getLastCrossNotarizedHeaders() []bootstrapStorage.BootstrapHeaderInfo
 }
 
-// OutGoingOperationsPool defines the behavior of a timed cache for outgoing operations
-type OutGoingOperationsPool interface {
```

Comment:
It was better to keep the interface definition at user/caller side, not where the implementation of the interface is.
Can stay as it is, though....